spec/functional/api/sessions_controller_spec.rb

```diff
@@ -78,6 +78,13 @@ module Pod::TrunkApp
       @owner.reload.name.should == 'Changed'
     end
 
+    it "does not update the owner's name in case it is missing on subsequent registrations" do
+      sign_in!
+      old_name = @owner.name
+      post '/', { 'email' => @owner.email, 'name' => ' ' }.to_json
```

Comment:
In #150 I removed a test that safe guarded against setting the name to a blank (nil or white space) string.